src/projects/shared/api/circuitdesigner/tests/helpers/CreateTestCircuitDesigner.ts

```diff
@@ -43,7 +43,9 @@ export function GetDefaultTools(): ToolConfig {
             SelectAllHandler, FitToScreenHandler, DuplicateHandler,
             DeleteHandler, SnipNodesHandler, DeselectAllHandler,
             SelectionHandler, SelectPathHandler, RedoHandler, UndoHandler,
-            CleanupHandler, CopyHandler, PasteHandler, ZoomHandler,
+            CleanupHandler, ZoomHandler,
+            // CopyHandler,
```

Comment:
Same as the other test helper, either delete the commented out Copy/PasteHandlers or leave a TODO for when they should be added back in